test/rendering.test.js

```diff
@@ -186,13 +187,13 @@ describe('Rendering', () => {
     } catch {
       // ignore
     }
+    console.log(expHtml);
     if (!expStatus) {
       // eslint-disable-next-line no-param-reassign
       expStatus = expHtml === null ? 404 : 200;
     }
     const response = await render(url, '', expStatus, partition);
     const actHtml = response.body;
-    console.log(actHtml);
```

Comment:
why was this removed? I like to keep a commented version of it for testing

Reply:
I accidentally removed this and kept the one I added. Fixed in 2489e05.